includes/amp-helper-functions.php

```diff
@@ -143,14 +141,22 @@ function () {
 	 */
 	$options     = get_option( AMP_Options_Manager::OPTION_NAME, [] );
 	$old_version = isset( $options[ Option::VERSION ] ) ? $options[ Option::VERSION ] : '0.0';
+
 	if ( AMP__VERSION !== $old_version && is_admin() && current_user_can( 'manage_options' ) ) {
-		/**
-		 * Triggers when after amp_init when the plugin version has updated.
-		 *
-		 * @param string $old_version Old version.
-		 */
-		do_action( 'amp_plugin_update', $old_version );
-		AMP_Options_Manager::update_option( Option::VERSION, AMP__VERSION );
+		// This waits to happen until after_setup_theme to ensure that amp theme support and amp post type support have all been added.
+		add_action(
+			'after_setup_theme',
```

Comment:
The `after_setup_theme` action would have already been fired, no? If I recall correctly, the `init` action comes after `after_setup_theme`.

Reply:
Humm, yeah, you're right. I need to re-check this.

Reply:
Fixed in 7df6a05378e8b7e6fdeefac9a159a6e7117257d3